.github/workflows/ci.yml

```diff
@@ -426,8 +457,13 @@ jobs:
             ~/Library/Caches/Coursier/v1
           key: ${{ runner.os }}-sbt-cache-v2-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('project/build.properties') }}
 
-      - name: Scalafix tests
-        if: matrix.scala == '2.13.8'
-        run: |
-          cd scalafix
-          sbt ci
+      - name: Generate site
+        run: sbt '++${{ matrix.scala }}' site/tlSite
+
+      - name: Publish site
+        if: github.event_name != 'pull_request' && github.ref == 'refs/heads/series/0.23'
```

Comment:
I wonder, wouldn't this break the publishing of the landing page? Or maybe I misapprehend how we publish it.

Reply:
Crappp this needs to be fixed :) should be publishing if main.

Anyway, the landing page is only published from main branch. The lower branches never publish versioned material.